src/relion/_parser/relion_pipeline.py

```diff
@@ -15,7 +15,7 @@
 
 
 class RelionPipeline:
-    def __init__(self, origin, graphin=ProcessGraph([])):
+    def __init__(self, origin, graphin=ProcessGraph([]), locklist=[]):
```

Comment:
[Suggest not using a mutable object as default value here](https://web.archive.org/web/20200221224620/http://effbot.org/zone/default-values.htm), as it can lead to interesting behaviour.

`locklist = None`
combined with
`self.locklist = locklist or []`
is the simplest way around this.